_component_design/tooltip.md

```diff
@@ -360,23 +357,16 @@ Hover tooltips are best used on large breakpoints and desktop designs. Tooltips
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Click interaction
-
-Use click enabled tooltips for smaller breakpoints and mobile designs. Tooltips can have both hover and click interactions on the same tooltip across breakpoints. Click enabled tooltips go through the following interaction lifecycle:
+## Loading state
 
-- **Hidden:** The base state of a tooltip is to be hidden until the trigger is clicked by the pointer.
-- **Displayed:** Tooltips enter the displayed state once the pointer has clicked the trigger element.
-- **Persist:** Clickable tooltips persist until manually removed by a user.
-- **Removal:** Tooltips re-enter the hidden state once the pointer has either:
-    - re-clicked the trigger element.
-    - clicked outside the tooltip.
+Apply a tooltip loading state exclusively to data visualizations.
```

Comment:
Will tooltips always use the same circular buffering icon for loading states? Can you use any loading state described in the "Loading States" pattern documentation in a tooltip?